src/routes/namespaces/[namespace]/workflows/[workflow]/[run]/events/events.d.ts

```diff
@@ -1,85 +1,85 @@
 type HistoryEvent =
-  import('$types/temporal/api/history/v1/message').HistoryEvent;
+  import('$types').HistoryEvent;
```

Comment:
Is this generated code?
Why do we need all of these separate import lines?

Reply:
afaik these event types may end up having custom properties or the same properties but custom mapping handlers

cc @stevekinney to correct me or provide details

Reply:
You might want to take the interfaces instead of the classes, I find them easier to work with.

```suggestion
  import('$types').IHistoryEvent;
```